apps/remix-ide/src/app/plugins/parser/code-parser.tsx

```diff
@@ -173,7 +171,7 @@ export class CodeParser extends Plugin {
      * @returns 
      */
     async getLastCompilationResult() {
-        return this.compilerAbstract
+        return await this.call('compilerArtefacts', 'get', '__last')
```

Comment:
that will not work. 'compilerArtefacts' doesn't have data from the parser compiler as 'compilerArtefacts' only listens to 'solidity' plugin which this isn't